docs/lua/functions-7.md

```
@@ -3646,6 +3646,27 @@ Skips animation interpolation for a frame
+`local integerValue = obj_get_count()`
+
+### Parameters
+- None
+
+### Returns
+- `integer`
+
+### C Prototype
+`u32 obj_get_count(void);`
+
+[:arrow_up_small:](#)
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)